src/mo_frontend/typing.ml

```diff
@@ -2434,7 +2445,7 @@ and infer_dec env dec : T.typ =
             display_typ_expand t'
             display_typ_expand t''
       | Some typ, T.Actor ->
-        local_error env dec.at "M0135" "actor class has non-async return type"
+         local_error env dec.at "M0135"(*FIXME: repeated?*) "actor class has non-async return type"
```

Comment:
This is not caught by CI as `M0135` already appears above, but the error string never occurs in the test suite.

Reply:
Good catch!